pkg/api/types.go

```diff
@@ -403,6 +404,13 @@ const (
 	PodUnknown PodPhase = "Unknown"
 )
 
+type PodCondition string
```

Comment:
My previous comment doesn't show here any more: I want this to look like NodeCondition.